package/MDAnalysis/core/groups.py

```diff
@@ -3054,8 +3054,7 @@ def write(self, filename=None, file_format="PDB",
         if filename is None:
             trjname, ext = os.path.splitext(os.path.basename(trj.filename))
             filename = filenamefmt.format(trjname=trjname, frame=trj.frame)
-        filename = util.filename(filename, ext=file_format.lower(), keep=True)
-
+        filename = util.filename(filename, ext= file_format or 'PDB', keep=True)
```

Comment:
PEP8 !

Reply:
Did you test that you can omit `.lower()`??

Reply:
I don't like `file_format or 'PDB'`. Better be explicit so that only `None` really gets defaulted:
```python
file_format if file_format is not None else 'PDB'
```

Reply:
> Did you test that you can omit `.lower()`??

If file_format is None. `.lower()` will throw an error, so I have shifted it inside `util.filename` function.